packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,945 @@
+  <ListItem>You need placeholder text in the field to show an example selection.</ListItem>
+  <ListItem>
+    Users would benefit from typeahead functionality (e.g., autocomplete, search). For example, typeahead may be useful
+    when users need to select from a list of more than 10 options. If you have a much longer list, consider lazy loading
+    the list to improve page load performance.
+  </ListItem>
+</UnorderedList>
+
+### Accessibility
+
+<UnorderedList>
+  <ListItem>
+    Include a visible label on <strong>all</strong> form fields.
+  </ListItem>
+  <ListItem>
+    Each lable must use the <inlineCode>htmlFor</inlineCode> prop that equals the <inlineCode>id</inlineCode> of the
+    associated input.
+  </ListItem>
+  <ListItem>
+    Don't use placeholder text as a replacement for labels. It can be used to provide an example to users, but will
+    disappear from the field when a user enters text. It's also not broadly supported by assistive technologies and
+    won't display in older browsers.
+  </ListItem>
+  <ListItem>
+    Use one of 3 ways to label a form field:
+    <UnorderedList marginBottom="space0">
+      <ListItem>
+        Visible label with <Anchor href="#formlabel-props">FormLabel</Anchor> (preferred)
+      </ListItem>
+      <ListItem>
+        Visible label that's associated to the input with <inlineCode>aria-labelledby</inlineCode>
+      </ListItem>
+      <ListItem>
+        Label directly using <inlineCode>aria-label</inlineCode>
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Provide clear identification of required fields in the label or at the start of a form. If you use the required
+    field indicator, include the form key at the start of the form.
+    <UnorderedList marginBottom="space0">
+      <ListItem>
+        Use the <inlineCode>required</inlineCode> prop to programatically indicate they are required to browsers.
+      </ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Include inline error text with the error icon on any field that errors to make it visually clear that the field
+    changed.
+  </ListItem>
+  <ListItem>
+    If the input has assicated help text or error text, include the <inlineCode>aria-describedby</inlineCode> prop on
+    the input. This should match the <inlineCode>id</inlineCode> of the help or error text.
+  </ListItem>
+</UnorderedList>
+
+## Examples
+
+### Base Select
+
+The base select is the input without any supporting elements (e.g., label, help text, etc).
+
+- **Options** — The list of items contained in a select
+- **Selected value** — (Optional) The value that appears in the select on page load. By default, this is the first option in the list.
+
+The select will fill the width of the container it’s placed in.
+
+<LivePreview scope={{Select, Option}} language="jsx">
+  {`<>
+  <Select id="console_team" htmlFor="console_team">
```

Comment:
Each select example should follow our best practices which are that you should include a visible label and use unique IDs